charts/dbildungs-iam-server/templates/cronjob.yaml

```diff
@@ -0,0 +1,89 @@
+{{- if .Values.cronjobs.enabled }}
+{{- range $job_name, $job_options := .Values.cronjobs.jobs }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "common.names.name" $ }}-{{ $job_name}}
+  namespace: {{ template "common.names.namespace" $ }}
+spec:
+  schedule: {{ $job_options.schedule }}
+  startingDeadlineSeconds: 300
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            cron: {{ $job_name }}
+        spec:
+          automountServiceAccountToken: false
+          containers:
+          - name: {{ $job_name }}
+            image: "{{ $.Values.cronjobs.image.repository }}:{{ $.Values.cronjobs.image.tag }}"
+            imagePullPolicy: {{ $.Values.cronjobs.image.pullPolicy | default "Always"}}
+            securityContext:
+              # not yet possible since we need to install some tools
```

Comment:
@simoncolincap since we install some stuff on the alpine image we need extra permissions, not sure whether to use a customized image with installed packages and disable root permissions